ethereum/spec/src/test/resources/tech/pegasys/teku/spec/config/bellatrixForkSpec.json

```diff
@@ -79,6 +79,7 @@
   "SLOTS_PER_EPOCH": "32",
   "SLOTS_PER_HISTORICAL_ROOT": "8192",
   "SYNC_COMMITTEE_SIZE": "512",
+  "SYNC_COMMITTEE_BRANCH_LENGTH": "5",
```

Comment:
Sadly we can't just update these test files (including `altairForkSpec.json` and `mergeForkSpec.json`). They're a capture of the actual response the beacon node will give from the /eth/v1/node/config endpoint which the standalone validator client uses to load the spec from the beacon node. It looks like adding this new property as a required field to Altair is breaking backwards compatibility (which is what the `SpecConfigLoaderTest.shouldLoadMainnetPreservingBackwardsCompatibilityWithRestApi` test is checking for.

We probably need to provide a default value in `SpecConfigBuilder` instead of requiring the field be set. Could just use the mainnet value as the default which I think would be reasonable. It's not particularly ideal but better than breaking backwards compatibility since people don't always update the VC and BN together. The default probably never gets used anyway since the VC doesn't use it and the built-in presets will provide it for the BN.

Reply:
Thanks @ajsutton I think that makes sense. I set a default in the builder, similar treatment as `updateTimeout`. To check my understanding, is there a point at which the validators are updated to receive the new configs? Or will we continue to use the default for some time?

Reply:
The validator client will load the config from the beacon node every time it starts up, so the real value should be picked up quite quickly but it's possible the VC is running against a beacon node that doesn't get updated or against a different client which doesn't yet have light client support and this config item.  So we'll probably wind up keeping the default around pretty much forever (or at least until the next hard fork if all clients include it by then since that forces everyone to update).  For the vast majority of setups though, they'll start loading the real value very quickly.